config/prow/config.yaml

```diff
@@ -195,6 +195,11 @@ tide:
     - do-not-merge/work-in-progress
     - needs-rebase
     - "cla: no"
+  - repos:
+    - gardener/gardener
+    milestone: v1.55
+    includedBranches:
+    - master
```

Comment:
Can you move this section up, so that it is next to the other g/g config?

I think we need to exclude `master` from the other g/g config. At least, that's what the kubernetes folks do: https://github.com/kubernetes/test-infra/commit/6c829da596d67eff62e0c3885992f6cc34ea251b
Also, we need the same `labels` and `missingLabels` sections. Otherwise, all PRs will be automerged as soon as they are added to the milestone independent of test results 😄

Reply:
I was hoping that Tide considers all the queries for the given repositories. Atm. the configs for `gardener/gardener` and `gardener/ci-infra` are the same. If my assumption is true, we would be able to keep it that way. Otherwise, we to split it up.
I'check this in the Tide codebase.

Reply:
@timebertt I split the config for `gardener/ci-infra` and `gardener/gardener`